dronegen/promote.go

```diff
@@ -54,7 +54,7 @@ func promoteBuildPipelines() []pipeline {
 }
 
 func publishReleasePipeline() pipeline {
-	p := relcliPipeline(triggerPromote, "publish-rlz", "Publish in Release API", "relcli auto_publish -f -v 6")
+	p := relcliPipeline(triggerPromote, "publish-rlz", "Publish in Release API", "auto_publish -f -v 6")
```

Comment:
This and "tag" invocation changed, as we now build a "distroless" image and entrypoint is set to `relcli`, so the "command" should consist of only the arguments to relcli.